data/json/monsters.json

```diff
@@ -2405,8 +2405,8 @@
   {
     "id": "mon_tankbot",
     "type": "MONSTER",
-    "name": "tank drone",
-    "description": "The Northrup Emancipator is the first and only automated tank ever produced, made shortly after the split-up of Northrup Grumman.  Clad in depleted uranium plating, and armed with advanced munitions and a 120-mm gun, it is capable of delivering extraordinary firepower.",
+    "name": "Beagle Mini-Tank UGV",
+    "description": "The Northrup Beagle is a tankette-sized urban warfare UGV. Sporting an anti-tank missile launcher, 40mm grenade launcher, and numerous anti-infantry weapons, it's designed for high-risk urban fighting.",
```

Comment:
Replace the end of the first sentence in the description with " warfare Urban Ground Vehicle (UGV). " There should also be two spaces after the '.' according to CDDA's manual of style.

Reply:
Isn't "Unmanned Ground Vehicle" the standard expansion of [UGV](https://www.google.ca/search?q=ugv&oq=ugv&aqs=chrome..69i57j0l5.535j0j7&sourceid=chrome&ie=UTF-8)? And I'll fix the typos.